test/Succeed/Issue4026.warn

```diff
@@ -1,4 +1,5 @@
 Issue4026.agda:1,1-30
+warning: -W[no]NotInScope
```

Comment:
This happens in `{-# COMPILE GHC myNat = 0 #-}`.  
Such warnings should tell the consequences, e.g. "Ignoring COMPILE pragma".
Maybe general form: "Ignoring declaration ... because ...".